CMakeLists.txt

```diff
@@ -68,6 +102,48 @@ target_link_libraries(
 option(BUILD_APPS "Build IceFlow examples" ON)
 option(BUILD_TESTS "Build library tests" OFF)
 
+configure_file(${IceFlow_SOURCE_DIR}/pkg/iceflow-config.cmake.in
+               ${IceFlow_BINARY_DIR}/pkg/iceflow-config.cmake @ONLY)
+
+configure_file(${IceFlow_SOURCE_DIR}/iceflow-config-version.cmake.in
+               ${IceFlow_BINARY_DIR}/iceflow-config-version.cmake @ONLY)
+
+install(FILES ${IceFlow_BINARY_DIR}/pkg/iceflow-config.cmake
+              ${IceFlow_BINARY_DIR}/iceflow-config-version.cmake
+        DESTINATION lib/iceflow-${version})
+
+target_include_directories(
+  iceflow
+  PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/iceflow>
+         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}${PROTO_BINARY_DIR}>)
```

Comment:
I'm not sure, but doesn't this expand to `${CMAKE_CURRENT_SOURCE_DIR}${CMAKE_CURRENT_BINARY_DIR}/generated` right now?

If so, this doesn't seem right, because both of these variables seem to be absolute paths.

Reply:
Yeah, seems like the line was in fact obsolete :) I think given that, we can resolve the thread :)